force-app/main/default/objects/JourneyTask__c/fields/Estimated_Completion_Date__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Estimated_Completion_Date__c</fullName>
+    <externalId>false</externalId>
+    <formula>Journey__r.Onboarding_Start_Date__c  +  TemplateTask__r.Relative_Days_to_Complete__c</formula>
```

Comment:
Wouldn't make sense to include a Relative Days To Complete under journey because Journey object records candidates/managers information and when they should begin. If the RDTC is there question is to complete..what

Reply:
Ok. So you've understood the initial data model and you've split the Journey tasks and Template tasks into their own separate objects. So they're independent of each-other. You've established some relationships but it makes me think you need some clarification on the concepts.

The idea is that Templates are blueprints for roles which detail tasks in order to pass a candidates probation. 
Journey's on the other hand are specific sets of tasks to complete for a _separate_ reasoning. 

So yes you are right to assume there's are relationships, but they need tweaking. If you have a Journey task, and an onboarding start date for that Journey. You want be able to calculate the estimated completion date relative to that Journey and it's Journey task. Referencing the template task's relative days to complete would give you the completion date based on the Journey and the unrelated template task. 

The Journey and Journey task objects are almost a copy but with extra steps.

Reply:
Complete